internal/commands/commands.go

```diff
@@ -66,22 +66,30 @@ Realm app will be exported as well.`,
 		Description: "Manage the apps associated with the current user",
 		Help:        "app help", // TODO(REALMC-7429): add help text description
 		SubCommands: []cli.CommandDefinition{
-			cli.CommandDefinition{
+			{
 				Use:         "init",
 				Aliases:     []string{"initialize"},
 				Display:     "app init",
 				Description: "Initialize a Realm app in your current local directory",
 				Help:        "",
 				Command:     &app.CommandInit{},
 			},
-			cli.CommandDefinition{
+			{
 				Use:         "list",
 				Aliases:     []string{"ls"},
 				Display:     "app list",
 				Description: "List the MongoDB Realm applications associated with the current user",
 				Help:        "list help", // TODO(REALMC-7429): add help text description
 				Command:     &app.CommandList{},
 			},
+			{
+				Use:         "diff",
+				Aliases:     []string{},
+				Display:     "app diff",
+				Description: "Show changes between a remote Realm application and the local directory associated with the current user",
+				Help:        "diff help", // TODO(REALMC-7429): add help text description
```

Comment:
Check out [this doc](https://docs.google.com/document/d/14fBZaN47HRe3-2Pk1uSfYcCxX3Dd95wK0tprS3WE1vk/edit#heading=h.clf8mfqw7ws8) that Kevin from design put together:

**Description:** `Show differences between your Realm app and your local directory`

**Help:** `Displays file-by-file differences between the latest version of your Realm app and your local directory. If you have more than one Realm app, you will be prompted to select a Realm app that you would like to display from a list of all Realm apps associated with your user profile.`

fwiw, i still need to go in to these help texts and clean it up/format them.  if you have any thoughts on how to do it well i'm all ears.  but the main crux of the issue here is we need to handle the line-width formatting ourselves for this long-form help text

visually speaking, it'd be pretty convenient to use go's backticks with multi-lines for this, not sure if there's a clever programmatic way about this as well